react/modalcolumns.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import Col from 'react-bootstrap/Col';
+import Button from 'react-bootstrap/Button';
+import { ReactComponent as EnvOk } from './assets/envelope_ok.svg';
+import { ReactComponent as EnvBugged } from './assets/envelope_bugged.svg';
+
+class ModalColumns extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+
+        }
+        this.chunkEnvelopesByBatchSize = this.chunkEnvelopesByBatchSize.bind(this);
+    }
+
+    chunkEnvelopesByBatchSize(batchSize, envelopes) {
```

Comment:
after looking at this again, maybe this would benefit from some comments to at least explain the idea and provide a sample input => output